src/core/Link/Link.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { BrowserRouter as Router, NavLink } from 'react-router-dom';
+import { useStyles } from './styles';
+
+interface LinksProps {
+  pathname: string;
+  disabled?: boolean;
+  target?: string;
+}
+
+const Links: React.FC<LinksProps> = ({
+  pathname,
+  target,
+  children,
+  disabled,
+}) => {
+  const classes = useStyles();
+  const disableLink = disabled ? classes.disabled : '';
+  return (
+    <Router>
```

Comment:
Router is not required the website using this...will implement it's own if needed

Reply:
@arkajyotiMukherjee 
As we are using react-router-dom, the router is required, otherwise it give issue leading to it not working. It's hardly for the website to be the issue.

Reply:
you can keep the router in storybook...but it'll not be in the component. The reason for that is link looks for the nearest router to it...so if it is wrapped in a router already it will be unable to router between the routes of the project it is used in.
@Hasmik8